### DISCUSSION TITLE
Github incorrect information for VGMTrans main page

### DISCUSSION THREAD

**Discussion by Moltresrider** [vgmtrans/vgmtrans#239] [Q&A] [answered]
The main page for VGMTrans says that there is a build available but the link only goes to a "workflow" page with NO download link whatsoever for the program.

Google does not offer any alternative download links and only lists this as the only relevant result for VGMTrans but there no option to download the program from Github unlike what the main page for VGMTrans says

**Comment by sykhro** [comment, marked as answer]
Click on whatever workflow, scroll down to artifacts, download the right build for your platform. GitHub doesn't provide a way to link to the latest build.

**Reply by Moltresrider** [reply]
There is NO option to

**Reply by Moltresrider** [reply]
The ONLY option is to "view workflow"

NO download links for ANY artifacts whatsoever.

**Reply by sykhro** [reply]
Stop being condescending. Every workflow has 3 download links.
Here's a screenshot from the latest run.
![Screenshot_20210818_235551](https://user-images.githubusercontent.com/8491400/129977554-03c9c505-0f27-400d-8c86-6db0158bb8fa.png)

**Reply by sykhro** [reply]
Here, at your convenience, the link for the [latest windows artifact](https://github.com/vgmtrans/vgmtrans/suites/3528720875/artifacts/84353202) found on the page for the [latest workflow run](https://github.com/vgmtrans/vgmtrans/actions/runs/1142417698)

**Reply by rebtd7** [reply]
The links only appear when logged in, which is confusing

**Reply by sykhro** [reply]
Complain with GitHub, nothing we can do about it. Best I could do is set up a page [here](https://vgmtrans.github.io/vgmtrans/)

**Comment by admin** [comment]
Now there's https://github.com/vgmtrans/vgmtrans-preview/releases/latest too
